addons/website_slides/controllers/main.py

```diff
@@ -841,6 +842,10 @@ def slide_preview(self, slide_id):
 
     @http.route(['/slides/slide/send_share_email'], type='json', auth='user', website=True)
     def slide_send_share_email(self, slide_id, email, fullscreen=False):
+        email_ids = email.split(",")
+        for email_id in email_ids:
```

Comment:
Why `email_id` ? it just a email isn't it ?
```suggestion
        for email in emails:
```

Reply:
Here we have conflict for name 'email' with function argument name. So, I have used mail and mails here.
    for mail in mails:

Reply:
the function argument should be `emails` it contain many emails :)

Reply:
Changes done.